app/src/main/java/org/listenbrainz/android/ui/components/TopBar.kt

```diff
@@ -82,12 +81,26 @@ fun TopBar(
         contentColor = MaterialTheme.colorScheme.onSurface,
         elevation = 0.dp,
         actions = {
-            IconButton(onClick = {
-                activity.startActivity(Intent(activity, AboutActivity::class.java))
+            // TODO: Move about-section to settings.
```

Comment:
Let's take this up in another PR and not this?

Reply:
Just commented it out for now, not doing anything with it.

Reply:
About is an important area. We can't simply comment it out. Either we replace it or have it on our radar in the next PR or two to fix it.

Reply:
Next PR or two would be good.